websites/P/Pomofocus/presence.ts

```diff
@@ -0,0 +1,46 @@
+const presence = new Presence({
+  clientId: '1350359904300564510',
+})
+
+const presenceData: PresenceData = {
+  details: 'Idling 💤',
+  state: '',
+  largeImageKey: 'https://i.ibb.co/TBYtYzMc/940956e1-8b78-424c-ad21-3a5e687dd6b4.jpg',
```

Comment:
Is this the correct logo?
```suggestion
  largeImageKey: 'https://i.ibb.co/spcHGMsY/undefined-Imgur-removebg-preview-512x512.png',
```

Reply:
The activity one was basically AI generated, and the logo containing a Right Tick is the original logo so far, I founded on the activity's site